planet/clients/subscriptions.py

```diff
@@ -119,7 +119,8 @@ async def cancel_subscription(self, subscription_id: str) -> None:
             APIError: on an API server error.
             ClientError: on a client error.
         """
-        url = f'https://api.planet.com/subscriptions/v1/{subscription_id}/cancel'
+        root_url = 'https://api.planet.com/subscriptions/v1'
```

Comment:
Consider having `ROOT_URL` as a global instead of declaring it twice?

Reply:
Yep. That will be done as a part of #628

Reply:
I'd prefer the root or base url be a SubscriptionsClient class attribute. It would have the advantages of a global (ease of patching for tests) but would also be scoped a little closer to the only place it is used - in client class methods.

Reply:
That's a great call

Reply:
It looks like the intent is to break a 81 character line into two lines less than 80 characters? I don't think it's worth doing, but Python's implicit string concatenation is probably better than adding a new variable. A future reader might mistake the extra variable assignment as being significant, when it's only incidental. Implicit string concatenation would go like this:

```python
url = ('https://api.planet.com/subscriptions/v1'
       f'{subscription_id}/cancel')
```

This would be even cleaner and tidier

```python
url = '{self.base_url}/{subscription_id}/cancel'
```

Reply:
these are good points and I fully agree with them. root_url is arbitrary and going to using `self.base_url` is preferred. I created #705 to address this and referenced this discussion.